test/test-lib/contracts-mock.ts

```diff
@@ -52,12 +52,16 @@ export const mockRepo = (
 	};
 };
 
-export const removeContractInterceptors = mockRepo(
-	{
-		owner: CONTRACTS_PUBLIC_REPO_OWNER,
-		name: CONTRACTS_PUBLIC_REPO_NAME,
-		branch: CONTRACTS_PUBLIC_REPO_BRANCH,
-	},
-	'base-contracts',
-	true,
-);
+export let removeContractInterceptors: () => void;
+export const addContractInterceptors = () => {
+	removeContractInterceptors = mockRepo(
+		{
+			owner: CONTRACTS_PUBLIC_REPO_OWNER,
+			name: CONTRACTS_PUBLIC_REPO_NAME,
+			branch: CONTRACTS_PUBLIC_REPO_BRANCH,
+		},
+		'base-contracts',
+		true,
+	);
+};
+addContractInterceptors();
```

Comment:
This also happens on first load the same as before, it's just it can be re-initialized at runtime the same as it is at loadtime so if you run the contract tests one after the other they actually reset the state properly (in master there's a somewhat janky reset that only partially does the thing - well enough to work in the context but not in a multi-version context)